### ISSUE TITLE
When a timeline start date is Sunday, the previous week shows up as an empty week 1

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#2360]

**Comment by Ace-Bansal** [comment]
@ragesoss  I'd like to work on this.

**Comment by tonynguyen111997** [comment]
Hi @Ace-Bansal, I noticed that this issue has been open for a month now and I couldn't find a PR on it. Are you still interested in working on this bug? If not I'd like to attempt to tackle it.

**Comment by ragesoss** [comment]
@tonynguyen111997 go for it. Let us know if you have any questions.

**Comment by Manasa2850** [comment]
@ragesoss can I work on this?

**Comment by ragesoss** [comment]
Go for it.

**Comment by Manasa2850** [comment]
@ragesoss could you please let me know where I can find the files related to this issue?

**Comment by ragesoss** [comment]
The place to start looking is `timeline_handler.jsx`. Most of the logic related to dates on the timeline is in `CourseDateUtils`, so that's where you'll probably end up.

**Comment by Manasa2850** [comment]
Ok. I'll have a look at it.

**Comment by abdelquodr** [comment]
Hi @ragesoss. Is anyone working on this ticket?

**Comment by ragesoss** [comment]
@abdelquodr no, no one is working on it. since it's been a few years, i recommend starting by confirming that the issue is still happening.

**Comment by nexpectArpit** [comment]
@ragesoss working on this!
